tests/secrets/sanity/secrets/true_positive.json

```diff
@@ -3,5 +3,6 @@
     "secret_key": "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
     "my_new_password": "F322a45xxmwov9bpgRhyuByXj2nxz7khS6yXQmfSaQCmwbTF2jpfgC56az3a",
     "test_pass": "z2b7k2cQfzc+yjP2K8cjuQ8uoorHBpEvC+XWhU3Z5+IdrPQYwr991Lj73xfZ+RA2GzC0wTedDTvb1C2NX+3Gpw==",
-    "pg_pass": "sup1rstr0ngpass2ForTT"
+    "pg_pass": "sup1rstr0ngpass2ForTT",
+    "secret_key": "THISISASECRETKEY1234=="
```

Comment:
is this related to the PR? 😄

Reply:
oops thought I removed that one, guess I haven't